app/models/king.rb

```diff
@@ -2,4 +2,15 @@ class King < Piece
   def to_unicode
     black? ? "\u265A" : "\u2654"
   end
+
+  def valid_move?(destination_x, destination_y)
+    origin_x = x_position
+    origin_y = y_position
+    distance_x = (destination_x - origin_x).abs
+    distance_y = (destination_y - origin_y).abs
+
+    return true if (0..1).cover?(distance_x) && (0..1).cover?(distance_y)
+
+    false
+  end
```

Comment:
Looks great!

One trick you could use.

Lines 12-14 could be reduced to a single line.

`(0..1).cover?(distance_x) && (0..1).cover?(distance_y)` will evaluate to `true` or `false`. Having it on the last line will act the same as the current code.